lib/phoenix/live_dashboard/components/nav_bar_component.ex

```diff
@@ -9,12 +9,16 @@ defmodule Phoenix.LiveDashboard.NavBarComponent do
   @impl true
   def update(assigns, socket) do
     %{page: page, items: items} = assigns
-    current = current_item(page.params, items)
-    {:ok, assign(socket, items: items, current: current, page: page)}
+    # TODO: document the usage of `nav_name`.
```

Comment:
Why do we need different nav_name? :)

Reply:
Oh, I got it. Shall we call it `nav_param`? Other than that, it looks good to me, we can make it public.

Reply:
Changed to `nav_param` and made it public.